### DISCUSSION TITLE
Stuck on "Ready To Connect" with QR Code

### DISCUSSION THREAD

**Discussion by alxjsn** [gtxaspec/wz_mini_hacks#635] [General]
Just leaving this for other people who are stuck. I tried everything and you know what worked?

Solution: Print the QR code on paper and then scan it and it will work immediately. I was flabbergasted.

**Comment by gtxaspec** [comment]
Turn the brightness on the phone so the way down.  That also works for me

**Comment by alxjsn** [comment]
I tried various brightness levels on both my matte finish thinkpad screen and my iPhone and for some reason I couldn't get it to work. Oh well.

**Comment by bjs-pdx** [comment]
Known issue for Wyze.  Several causes and solutions.  One problem, due to screen layout, is that the entire code isn't visible onscreen.  Using a printed version fixes several issues from illegibility to missing parts of the code.
